website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -130,15 +130,24 @@ to distribute Client connections across them.
 
 ### Performance tuning
 
-The default receive buffer size on Linux is quite small which can limit the
-maximum throughput that users perceive in "upload scenarios" (i.e. where the
-Gateway needs to receive large volumes of traffic).
-
-On startup, the Gateway attempts to increase the size of the UDP receive buffers
-to 10 MB. However, the actual size of the receive buffer is limited by the
-`net.core.rmem_max` kernel parameter. For the increased buffer size to take
-effect, you may need to increase the `net.core.rmem_max` parameter on the
-Gateway's host system.
+The default receive and send buffer sizes on Linux are quite small which can
+limit the maximum throughput that users can achieve.
+
+The Gateway attempts to increase the UDP send and receive buffer sizes on the
+system to 16 and 128 MB respectively. It does that by setting
```

Comment:
```suggestion
The Gateway attempts to increase the *system-wide* UDP send and receive buffer max sizes to 16 and 128 MB respectively. It does that by setting
```
suggestion: 
-  make it more clear that it's a system-wide setting that can impact other processes
- be clear that it's just the "max" sizes, not the default